nipap-cli/nipap

```diff
@@ -74,7 +74,12 @@ if __name__ == '__main__':
         sys.exit(1)
 
     # read configuration
-    cfg = ConfigParser.ConfigParser()
+    cfg_defaults = {
+        'scheme': 'http',
```

Comment:
the word scheme wasn't pulled out of thin air: http://www.iana.org/assignments/uri-schemes/uri-schemes.xhtml  It's part of a URI spec.

Since the current config is based off the URI parts and not a class constructor I say we stick to the international standards for now until we wrap up the initialization into something else.